src/client/components/collection/CollectionCard.js

```diff
@@ -149,14 +145,15 @@ export const CollectionCard = ({ collection = {} }) => {
           content: classes.content,
         }}
       />
-      <CardActionArea onClick={handleCollectionClick}>
+      <Link to={`/collections/${id}`}>
+        {/*<CardActionArea onClick={handleCollectionClick}>*/}
```

Comment:
I think this component comes with a highlight when hovering the children, I don't know if we should use this component and `history`'s `push`, or `Link` as you did.